.gitlab-ci.yml

```diff
@@ -6,7 +6,7 @@ cache:
 variables:
   AKKA_TEST_TIMEFACTOR: 10
   COURSIER_CACHE: "${CI_PROJECT_DIR}/project/.sbt/coursier"
-  SBT_TASK_LIMIT: 4
+  SBT_OPTS: "-XX:+CMSClassUnloadingEnabled  -XX:+UseConcMarkSweepGC -Xss2m -Xms3g -Xmx3g"
```

Comment:
no need, we have a `.jvmopts` file

Reply:
Yeah i saw it, it's mostly a test at that point.